contracts/modules/LoanOpenings/LoanOpenings.sol

```diff
@@ -70,160 +56,89 @@ contract LoanOpenings is State, LoanOpeningsEvents, VaultController, InterestHan
         }
 
         // get required collateral
-        uint256 collateralAmountRequired = _getRequiredCollateral(
-            loanParamsLocal.loanToken,
-            loanParamsLocal.collateralToken,
-            sentValues[1],
-            initialMargin,
-            isTorqueLoan
-        );
+        uint256 collateralAmountRequired = _getRequiredCollateral(loanParamsLocal.loanToken, loanParamsLocal.collateralToken, sentValues[1], initialMargin, isTorqueLoan);
         require(collateralAmountRequired != 0, "collateral is 0");
 
-        return _borrowOrTrade(
-            loanParamsLocal,
-            loanId,
-            isTorqueLoan,
-            collateralAmountRequired,
-            initialMargin,
-            sentAddresses,
-            sentValues,
-            loanDataBytes
-        );
+        return _borrowOrTrade(loanParamsLocal, loanId, isTorqueLoan, collateralAmountRequired, initialMargin, sentAddresses, sentValues, loanDataBytes);
     }
 
-    function setDelegatedManager(
-        bytes32 loanId,
-        address delegated,
-        bool toggle)
-        external
-        pausable
-    {
-        require(loans[loanId].borrower == msg.sender, "unauthorized");
-
-        _setDelegatedManager(
-            loanId,
-            msg.sender,
-            delegated,
-            toggle
-        );
+    function getDefaultLoanParamsAndStore(
```

Comment:
this is a view only function. which will take any loan/collateral addy and return them back
this function doesn't sanitize input. however in conjunction with how its currently used its impossible for it to be called in protocol context with the wrong arguments. 
question to the reviewer:  should this function sanitize input? e.g check `supportedTokens` for example for valid loan/collateral addy.